docs/pubsub-usage.rst

```diff
@@ -45,17 +60,9 @@ List topics for the default project:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import list_topics
-   >>> topics, next_page_token = list_topics()  # API request
-   >>> [topic.name for topic in topics]
-   ['topic_name']
-
-List topics for an explicit project:
-
-.. doctest::
-
-   >>> from gcloud.pubsub import list_topics
-   >>> topics, next_page_token = list_topics(project='my.project')  # API request
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topics, next_page_token = client.list_topics()  # API request
```

Comment:
Longer discussion over on #895 -- I think we decided last time to not discuss that as part of the PR, but wanted your opinion regardless.